### DISCUSSION TITLE
How to create multilingual dashboards with Quarto?

### DISCUSSION THREAD

**Discussion by sbacelar** [quarto-dev/quarto-cli#10550] [Q&A]
### Description

Is there a recommended solution for creating a multilingual dashboard with Quarto? I read some solutions for websites and books using `profiles` or `babelquarto`. Is the process similar to that of dashboards, considering that the dashboards will be published on the web?

**Comment by mcanouil** [comment]
That's completely up to you to first figure a way to have a multilingual dashboard which likely involves making several dashboards for example using profiles.

The "recommended" way is the way that works for you.

**Reply by sbacelar** [reply]
Can you give any (toy) example of a multilingual dashboard using profiles?

**Reply by mcanouil** [reply]
No, because I am not aware of anyone that did and shared this.
